src/api/http/httpRequest.ts

```diff
@@ -8,8 +8,8 @@ export class HttpRequest {
     this.service = service;
   }
 
-  get(url: ApiUrlType) {
-    return this.service.get(url).catch((error) => {
+  get(url: ApiUrlType, queryString: string = '') {
```

Comment:
const, 요런 '' 은 타입 추론이 됩니다!
```suggestion
  get(url: ApiUrlType, queryString = '') {
```